packages/cli/src/index.test.ts

```diff
@@ -0,0 +1,5 @@
+import { assert, expect, it, suite, test } from 'vitest';
```

Comment:
@lascenify we'll use Vite extensively from Gestalt so I thought it'd be a good idea to use tooling that builds on it. [Vitest](https://vitest.dev) is a testing framework that's becoming very popular lately. Which one are you using at your current company?